### ISSUE TITLE
MSColab Changeset view confusing/untidy

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#495]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/495) by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

The changeset view shows the diffs of the changes, which are typically rather large. The requires a lot of scrolling. And the diffs are not particular helpful.

A list with change times/abstracted info \(i.e. added point moved point/etc, but time would be fine for now\) might be better, where a double-click on the item \(or similar\) would show the full change?

The simplest undo button simply rolls back the lastest change. One might move the current stuff to a sub menu and offer “just” an undo button on the main window.

**Comment by ayeankit** [comment]
Hello @ReimarBauer , Is this still open?

**Comment by ReimarBauer** [comment]
Yes it is still open, but it would be too small to be a standalone gsoc project. If you want work on this, think about a solution without removing functionality.
